go/client/logclient.go

```diff
@@ -37,6 +37,7 @@ type LogClient struct {
 	uri        string                // the base URI of the log. e.g. http://ct.googleapis/pilot
 	httpClient *http.Client          // used to interact with the log via HTTP
 	verifier   *ct.SignatureVerifier // nil if no public key for log available
+	logger     Logger
```

Comment:
I know it will probably seem self-evident, but can you add a post-fix comment for this field to be consistent with the docs above.